openedx/core/djangoapps/schedules/tests/test_content_highlights.py

```diff
@@ -0,0 +1,112 @@
+# -*- coding: utf-8 -*-
+from openedx.core.djangoapps.schedules.config import COURSE_UPDATE_WAFFLE_FLAG
+from openedx.core.djangoapps.schedules.content_highlights import get_week_highlights, course_has_highlights
+from openedx.core.djangoapps.schedules.exceptions import CourseUpdateDoesNotExist
+from openedx.core.djangolib.testing.utils import skip_unless_lms
+from openedx.core.djangoapps.waffle_utils.testutils import override_waffle_flag
+
+from xmodule.modulestore.tests.django_utils import TEST_DATA_SPLIT_MODULESTORE, ModuleStoreTestCase
+from xmodule.modulestore.tests.factories import CourseFactory, ItemFactory
+from student.tests.factories import UserFactory
+from student.models import CourseEnrollment
+
+
+@skip_unless_lms
+class TestContentHighlights(ModuleStoreTestCase):
+    MODULESTORE = TEST_DATA_SPLIT_MODULESTORE
+
+    def setUp(self):
+        super(TestContentHighlights, self).setUp()
+        self._setup_course()
+        self._setup_user()
+
+    def _setup_course(self):
+        self.course = CourseFactory.create()
+        self.course_key = self.course.id
+
+    def _setup_user(self):
+        self.user = UserFactory.create()
+        CourseEnrollment.enroll(self.user, self.course_key)
+
+    def _create_chapter(self, **kwargs):
+        ItemFactory.create(
+            parent=self.course,
+            category='chapter',
+            **kwargs
+        )
+
+    @override_waffle_flag(COURSE_UPDATE_WAFFLE_FLAG, True)
+    def test_non_existent_course(self):
```

Comment:
"test_getting_highlights_for_non_existent_course_throws_exception"

Reply:
Nimisha, what do you think if we named tests differently? We can orient the reader and improve skimming if we name the tests as short sentences that declare what they enforce. I'll offer a few suggestions. Feel free to disregard if you don't think it helps.

Reply:
updated to `test_non_existent_course_raises_exception`.